test/integration/project_test.go

```diff
@@ -101,18 +100,22 @@ func TestProjectCreate_ApplicationDoesntExist_FailAndReturnErrorMessage(t *testi
 		flag(params.ApplicationName), "application-that-doesnt-exist",
 	)
 
```

Comment:
add test to get by name, with and without exact match

Reply:
irrelevant after last change